javascript-functions/main.js

```diff
@@ -0,0 +1,33 @@
+function convertMinutesToSeconds(minutes) {
+  var seconds = minutes * 60;
+  return seconds;
+}
+var convertMinutesToSecondsResult = convertMinutesToSeconds(5);
+console.log('There are' + ' ' + convertMinutesToSecondsResult + ' ' + 'seconds');
+
+function greet(name) {
+  return 'Hey,' + ' ' + name;
+}
+var greetResult = greet('Mike');
+console.log('result:', greetResult);
+
+function getArea(width, height) {
+  var area = width * height;
+  return area;
+}
+var getAreaResult = getArea(2, 3);
+console.log('The area is', getAreaResult);
+
+function getFirstName(person) {
+  var firstName = person.firstName;
+  return firstName;
+}
+var getFirstNameResult = getFirstName({ firstName: 'Sand', lastName: 'Wich' });
+console.log('The first name is', getFirstNameResult);
+
+function getLastElement(array) {
+  var lastEl = array.splice(array.length - 1);
```

Comment:
Oh no!!! This will remove the item from the array entirely. Now whenever this function gets called to see what the last item is, that item ceases to exist. You should just look up the value, not remove it from the array.